entgql/internal/todoplugin/todo.graphql

```diff
@@ -0,0 +1,86 @@
+directive @someDirective(
```

Comment:
Is it generated or written by hand?

Reply:
It's written by hand to apply on GQL object. I will remove it for now.